### ISSUE TITLE
DeprecationWarning

### ISSUE THREAD

**Issue by tavurth** [catamphetamine/universal-webpack#93]
```
(node:25740) DeprecationWarning: Tapable.plugin is deprecated. Use new API on `.hooks` instead
    at Chunk_file_names_plugin.apply (/home/tavurth/projects/frontend/node_modules/universal-webpack/build/chunks plugin.js:52:11)
    at webpack (project/node_modules/webpack/lib/webpack.js:37:12)
    at getOptions.then (project/node_modules/webpack-serve/index.js:26:30)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
    at Function.Module.runMain (module.js:695:11)
    at startup (bootstrap_node.js:188:16)
    at bootstrap_node.js:609:3
```

**Comment by catamphetamine** [comment]
Actually, yeah, I think the time has come to deal with it.

**Comment by catamphetamine** [comment]
Fixed in `0.6.6`.
